pkg/operator/ceph/config/datapath.go

```diff
@@ -0,0 +1,50 @@
+package config
+
+import "path"
+
+// A DataPathMap is a struct which contains information about where Ceph daemon data is stored in
+// containers and whether the data should be persisted to the host. If it is persisted to the host,
+// directory on the host where the specific daemon's data is stored is given.
+type DataPathMap struct {
```

Comment:
I think this struct is really helpful for being able to reason about how paths in the container map to paths on the host. I expect this to be more helpful for osds.